apps/shared-components/badge/badge.component.html

```diff
@@ -8,7 +8,10 @@
   class="{{ position }} {{ color }}"
 >
   <div>
-    <p [ngClass]="{ 'small-text': fontSize == 'small' }" class="{{ fontColor }}">
+    <p
+      [ngClass]="{ 'small-text': fontSize == 'small', 'extra-small-text': fontSize == 'extraSmall' }"
+      class="{{ fontColor }}"
+    >
       <nb-icon *ngIf="nbIcon" icon="{{ nbIcon }}"></nb-icon>
```

Comment:
This will make a conflict , on another PR you name it as mini in same component and here it is extrasmall

And name should be extra-small or xs